sentry_sdk/tracing.py

```diff
@@ -413,6 +418,91 @@ def get_trace_context(self):
         return rv
 
 
+class Transaction(Span):
+    __slots__ = ("name",)
```

Comment:
nit: No need for the tuple right?

```suggestion
    __slots__ = "name"
```

Reply:
I am surprised to learn that works but I would like to avoid it.